src/ConcordiumNetSdk/Helpers/Serialization.cs

```diff
@@ -0,0 +1,57 @@
+using System.Buffers.Binary;
+
+namespace ConcordiumNetSdk.Helpers;
+
+/// <summary>
+/// Helpers for serializing data.
+/// </summary>
+public class Serialization
```

Comment:
Nitpicky but, 

I don't see much value from having this class as functions as is. You're just guarding the function calls between one boolean, and as far as I know then for the current use case the sdk will only use big-endian as there are no smart contract schema support. 

Secondly, there is no test for it as far as I can see which I believe there should be.

Reply:
Yea, the only reason I did it is because this was done in the previous codebase, so I thought there may was a use for it.

I will add the tests in subsequent PR.